### DISCUSSION TITLE
installing-vban-on-linux-with-pipewire

### DISCUSSION THREAD

**Discussion by giscus** [engineerjoe440/stanley-solutions-blog#10] [Comments]
# installing-vban-on-linux-with-pipewire

I recently decided to drop Windows (finally), and move to Linux, full-time. But that means moving to Pipewire; which is both exciting and slightly daugnting. You see, to make the move, I need to get VBAN working on Linux, and talking to Pipewire. Hmm. Time to do some digging...

https://blog.stanleysolutionsnw.com/installing-vban-on-linux-with-pipewire.html

**Comment by d1scolor** [comment]
PipeWire has a vban-send modue and a vban-recv modue, I wonder if they can be used instead of installing vban from source.

**Reply by engineerjoe440** [reply]
Holy smokes! I didn't realize this was available! WOW!!! That'll be SOOOO much better than installing from source, and would certainly make integration much easier. Right now, I've mostly been using a FIFO to make everything work. A module in PW would be so much better.

**Reply by TechaNima** [reply]
So update soon™? Pretty please? I've been trying to get this damn thing to work as well and I got about as far as you did. I can't find any instructions on how to use PipeWire directly with vban-send. I have no idea what to do with the scraps of info that I have found. I tried with PulseAudio on Debian 11 and once I had properly wrecked the audio on that install without results, I installed Debian 12 and Jack. Still no dice. Doesn't help that every Jack turorial is like 5-10 years old and only a fork of vban by quiniouben has Jack support at all. ChatGPT was completely useless as well.
